src/astarte-client/client.ts

```diff
@@ -20,7 +20,7 @@ import axios from 'axios';
 import { Socket as PhoenixSocket } from 'phoenix';
 import _ from 'lodash';
 
-import { fromAstartePipelineDTO, toAstartePipelineDTO } from './transforms';
+import { fromAstarteDeviceDTO, fromAstartePipelineDTO, toAstartePipelineDTO } from './transforms';
```

Comment:
what about XfromY? (like deviceFromDTO)

Reply:
Since we have Typescript I think it's clear what that function is returning. However we can discuss better naming but I'd do the conversion in another PR where we change the naming to all transforms, for uniformity

Reply:
yea, we will probably need a different PR for that